class.jetpack-gutenberg.php

```diff
@@ -326,6 +326,20 @@ public static function get_jetpack_gutenberg_extensions_whitelist() {
 		return $preset_extensions;
 	}
 
+	/**
+	 * Returns a diff from a combined list of whitelisted extensions and extensions determined to be excluded
+	 *
+	 * @return array A list of blocks: eg [ 'publicize', 'markdown' ]
+	 */
+	public static function get_available_extensions() {
+		$exclusions             = get_option( 'jetpack_extensions_exclusions', array() );
+		$whitelisted_extensions = self::get_jetpack_gutenberg_extensions_whitelist();
+
+		$combined_extension_list = array_diff( $whitelisted_extensions, $exclusions );
+
+		return $combined_extension_list;
```

Comment:
This could save setting these three redundant variables:

```php
return array_diff(
    self::get_jetpack_gutenberg_extensions_whitelist(),
    get_option( 'jetpack_extensions_exclusions', array() )
);
```

Reply:
Good catch and thanks. Have updated this to just return the diff